### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by manzitresor**
This sections contains Dynamic project details created using javascript objects

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -589,4 +688,53 @@ hr {
...
+  .popup-content {
+    width: 1156px;
+    height: 570px;
+    border-radius: 24px;
+    display: block;
+    gap: 20px;
+    padding: 0 auto;
+    margin: 20px auto;
+    overflow-y: scroll;
+  }
```
Comment (AlphaNtihinduka) [inline styles.css]: - [ ] I have noticed that your `popup-content` has a fixed width. This breaks the responsiveness of the popup in different screen sizes. Kindly consider using the percentage width to match the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=u9NC5n6sAS6bpo65-0) 😎
- [ ] In addition, kindly ensure that the height of the popup content matches the Figma design. You may need to consider reducing the height accordingly ☺️.

Your design at 900px
image

Figma design 😎 
image

File: styles.css [hunk 2]
```diff
@@ -116,12 +120,100 @@ body {
...
+  height: 100vh;
+  background-color: #fff;
+  align-items: center;
+  padding: 12px 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin-left: auto;
+  margin-right: auto;
+  overflow-y: scroll;
+}
```
Comment (AlphaNtihinduka) [inline styles.css]: - [ ] I have noticed that your `popup-content` has a fixed width. This breaks the responsiveness of the popup in different screen sizes. Kindly consider using the percentage width to match the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=u9NC5n6sAS6bpo65-0) 😎
- [ ] In addition, kindly ensure that the height of the popup content matches the Figma design. You may need to consider reducing the height accordingly ☺️.

Your design 😎 
image

Figma design 😎 
image

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hello @manzitresor 🙋‍♂️
Good job working on your window popup. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement a few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- The use of an array of objects to store the project contents ✔️ 
- Correct use of forEach method ✔️ 

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or Slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @manzitresor 🙋‍♀️

Great work implementing the changes suggested by the previous reviewer 💪**_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Commits are descriptive, good work.
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**